pkg/storaged/filesystem/utils.jsx

```diff
@@ -174,7 +185,23 @@
                     </StorageLink>
                 </FlexItem>
             </Flex>
-            }
             { extra_text }
         </>);
 };
+
+export const mount_point_text = (mount_point, mounted) => {
+    let mp_text;
+    if (mount_point) {
+        mp_text = client.strip_mount_point_prefix(mount_point);
+        if (mp_text == false)
+            return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19807-20240103-141207-4bbc74c3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)